Objects/listobject.c

```diff
@@ -2145,12 +2145,21 @@ list.sort
     key as keyfunc: object = None
     reverse: bool(accept={int}) = False
 
-Stable sort *IN PLACE*.
+Sort the list in ascending order.
+
+The sort is in-place (i.e. the list itself is modified) and stable (i.e. the
```

Comment:
With the return specified as None, the sort must mutate the list, and 'in-place' can go.

Reply:
I think the "in-place" is essential and should be emphasized.  Formerly, we had this in all caps and surrounded by asterisks.

Reply:
OK.